.pre-commit-config.yaml

```diff
@@ -34,18 +34,15 @@ repos:
     rev: v0.942
     hooks:
       - id: mypy
-        args:
-          - --ignore-missing-imports
-          # Silence errors about Python 3.9-style delayed type annotations on Python 3.8
-          - --python-version
-          - "3.9"
+        args: []
         additional_dependencies:
           # Type stubs
           - types-docutils
           - types-requests
           - types-paramiko
           - types-PyYAML
           - types-psutil
+          - types-setuptools
```

Comment:
Revert #6017 - this is still used by mypy to validate setup.py (you can see it by setting ``ignore_missing_imports = false`` below)